src/app/course-page/course-page.component.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../styles/main.scss';
```

Comment:
better to use the absolute path here, so if you change the component folder it won't break
Also, import just variables. You don't need the whole `main.scss` file
`@import 'src/styles/variables';`

Reply:
done